src/styles/link-card-styles.scss

```diff
@@ -0,0 +1,26 @@
+.link-card {
```

Comment:
nit: this file can just be named link-card, consistent with the other scss files in the folder